aws/cluster/templates/cluster-spec.yaml

```diff
@@ -33,6 +33,7 @@ ${etcd-clusters}
     - DefaultTolerationSeconds
     - NodeRestriction
     - Priority
+    - DenyEscalatingExec
```

Comment:
This seems to be [deprecated](https://github.com/dataiku/dku-kubernetes/blob/master/CHANGELOG-1.14.md)?

Reply:
Yes, deprecated but not gone. Still need to do the work on its replacement so it stays for now.